File: Programming-languages-and-laboratory/Java/Exercises/36_compito_java_2020_01_16/Cittadino.java
public class Cittadino extends Supermercato {
	private int orarioApertura, orarioChiusura;
	private float superficie;
	private int numeroAddetti;
	private int numeroVisite;

	public Cittadino(int codice, String tipo, String indirizzo, int orarioApertura, int orarioChiusura,
			float superficie, int numeroAddetti) {
		super(codice, tipo, indirizzo);
		this.orarioApertura = orarioApertura;
		this.superficie = superficie;
		this.numeroAddetti = numeroAddetti;
		numeroVisite = 0;
	}

	public void addvisite() {
		numeroVisite++;
	}

	public int getNumeroVisite() {
		return numeroVisite;
	}

	public String toString() {
		return codice + "\t" + tipo + "\t" + indirizzo + "\t" + orarioApertura + "\t" + orarioChiusura + "\t"
				+ superficie + "\t" + numeroAddetti + "\t-";
	}
}

File: II - Linguaggi di programmazione e laboratorio/Python/Exercises/36_compito_python_20200116/Cittadino.py
from Supermercato import Supermercato


class Cittadino(Supermercato):
    def __init__(self, codice, tipo, indirizzo, apertura, chiusura, superficie, addetti):
        super().__init__(codice, tipo, indirizzo, superficie, addetti)
        self._apertura = apertura
        self._chiusura = chiusura

    def __str__(self):
        return str(self._codice) + "\t" + self._tipo + "\t" + self._indirizzo + "\t" + str(self._apertura) + "\t" +\
               str(self._chiusura) + "\t" + str(self._superficie) + "\t" + str(self._addetti) + "\t-"
    